schema/artwork/layers.js

```diff
@@ -0,0 +1,18 @@
+import { remove } from 'lodash';
+import gravity from '../../lib/loaders/gravity';
+import { enhance } from '../../lib/helpers';
+import ArtworkLayer from './layer';
+import { GraphQLList } from 'graphql';
+
+export const artworkLayers = id =>
+  gravity(`related/layers`, { artwork: [id] })
+    .then(layers => enhance(layers, { artwork_id: id }))
+    .then(layers =>
+      // Move fair layer to the beginning
```

Comment:
I suppose this is weird that I'm doing it in place like this. `remove` will return an array of the removed items. That then  gets concatted with the mutated array, effectively shifting the fair to the beginning.